src/lib/date.ts

```diff
@@ -4,7 +4,11 @@ const formattedDateStatusDate = date => {
   const momentToUse = moment.utc(date, "YYYY-MM-DD[T]HH:mm:ss")
   const momentDate = momentToUse.format("MMM D")
   const momentHour = momentToUse.format("ha")
-  if (!!momentHour && !!momentDate) {
+  const momentMinutes = momentToUse.format("mm")
+  const momentHourWithMinutes = momentToUse.format("h:mma")
+  if (!!momentHour && !!momentDate && momentMinutes !== "00") {
```

Comment:
There’s no need to format a string for the minutes check, could just do `momentMinutes.minutes() !== 0`.

I’d also like to move the `momentHourWithMinutes` formatting _inside_ the matching `if` branch so it isn’t done unneeded.

Reply:
Btw, is there a good reason for the double bang? Afaik we only need that when we need to pass a boolean value, not when performing a conditional check.

Reply:
Cleaned this section up.